src/lib/converters/options/computedConverter.ts

```diff
@@ -6,6 +6,78 @@ import {
   storePath,
 } from "../../helper";
 
+const mapArrayConverter = (
+  mapName: string,
+  namespaceText: string,
+  mapArray: ts.ArrayLiteralExpression
+) => {
+  const names = mapArray.elements as ts.NodeArray<ts.StringLiteral>;
+
+  switch (mapName) {
+    case "mapState":
+      return names.map(({ text: name }) => {
+        return {
+          use: "computed",
+          expression: `const ${name} = computed(() => ${storePath}.state.${namespaceText}.${name})`,
+          returnNames: [name],
+        };
+      });
+    case "mapGetters":
+      return names.map(({ text: name }) => {
+        return {
+          use: "computed",
+          expression: `const ${name} = computed(() => ${storePath}.getters['${namespaceText}/${name}'])`,
+          returnNames: [name],
+        };
+      });
+  }
+};
+
+const mapObjectConverter = (
+  mapName: string,
+  namespaceText: string,
+  mapObject: ts.ObjectLiteralExpression
+) => {
+  const props = mapObject.properties as ts.NodeArray<ts.PropertyAssignment>;
+
+  return props
+    .map((prop) => {
+      const name = prop.name as ts.Identifier;
+      const initializer = prop.initializer;
+
+      // function values are not currently supported.
+      if (
+        ts.isFunctionExpression(initializer) ||
+        ts.isArrowFunction(initializer)
+      ) {
+        throw new Error(
+          "Function value in a map object is not currently supported."
+        );
+      }
```

Comment:
Due to lack of time, the case where the value of an object is a function is not done. I will do it if the PR is merged.